homeassistant/components/sensor/yweather.py

```diff
@@ -138,7 +138,9 @@ def device_state_attributes(self):
 
     def update(self):
         """Get the latest data from Yahoo! and updates the states."""
-        self._data.update()
+        if not self._data.update():
```

Comment:
This will happen a lot because you're throttling the `update()` call. That means while it is being throttled, it will return `None`